flink-python/setup.py

```diff
@@ -43,15 +42,47 @@ def remove_if_exists(file_path):
             rmtree(file_path)
 
 
-def find_file_path(pattern):
-    files = glob.glob(pattern)
-    if len(files) < 1:
-        print("Failed to find the file %s." % pattern)
-        exit(-1)
-    if len(files) > 1:
-        print("The file pattern %s is ambiguous: %s" % (pattern, files))
-        exit(-1)
-    return files[0]
+def copy_files(src_paths, output_directory):
+    for src_path, file_mode in src_paths:
+        if os.path.isdir(src_path):
+            child_files = os.listdir(src_path)
+            for child_file in child_files:
+                dst_path = copy(os.path.join(src_path, child_file), output_directory)
+                os.chmod(dst_path, file_mode)
+        else:
+            dst_path = copy(src_path, os.path.join(output_directory, os.path.basename(src_path)))
+            os.chmod(dst_path, file_mode)
+
+
+def extracted_output_files(base_dir, file_path, output_directory):
+    extracted_file_paths = []
+    from xml.dom.minidom import parse
+    dom = parse(file_path)
+    root_data = dom.documentElement
+    file_elements = (root_data.getElementsByTagName("files")[0]).getElementsByTagName("file")
+    # extracted <files><file></file></files>
+    for file_element in file_elements:
+        source = ((file_element.getElementsByTagName('source')[0]).childNodes[0]).data
```

Comment:
Currently this function does not support the `<includes></includes>` tag and `<excludes></excludes>` tag. I think we need to raise Exception when the file element contains them to ensure the correctness.